### ISSUE TITLE
Verify if old versions can work despite new download strategies in dictionaries

### ISSUE THREAD

**Issue by rhadamanthe** [rhadamanthe/host-grabber-pp#72]
#71 has introduced a new strategy to find media to download.
It means dictionaries can use it. But will old versions of the extension be able to work with such an entry?

It should be verified.
If there is a problem, something should be done so that it does not arise again in the future.

**Comment by tzr7** [comment]
I can test this tonight (CEST) if it helps.

**Comment by tzr7** [comment]
I installed version 0.8 and used my hosts.xml with the imagevenue css query.
In the Debug-View I selected the entry. The "Search Pattern" showed up as undefined.
A click on "Extract Links" on a page with two imagevenue images matching domain and path pattern showed tow "undefined" at the bottom.
![hostgrabberpp_debug](https://user-images.githubusercontent.com/39011346/79987387-a0c9a680-84ad-11ea-919b-32dace2b5a92.png)

"Extract and Download" from the context menu on the page with on a page with two imagevenue images matching domain and path pattern did not discover any any links. No error shown.

I tested a couple of other pages with different image hosts and they all worked.

Seems fine to me.
